pkg/services/ngalert/state/manager.go

```diff
@@ -211,3 +221,46 @@ func translateInstanceState(state ngModels.InstanceStateType) eval.State {
 		return eval.Error
 	}
 }
+
+func (st *Manager) createAlertAnnotation(new eval.State, alertRule *ngModels.AlertRule, result eval.Result) {
+	st.log.Debug("alert state changed creating annotation", "alertRuleUID", alertRule.UID, "newState", new.String())
```

Comment:
Maybe this will be too noisy, given annotation appear in the UI.

Reply:
I think because it's debug logging it should be ok. The intention is to be able to troubleshoot missing annotations that also do not result in an error.